CMakeLists.txt

```diff
@@ -27,9 +27,10 @@ include(LXQtTranslate)
 
 add_subdirectory(src)
 add_subdirectory(liblxqt-config-cursor)
-add_subdirectory(lxqt-config-input)
-add_subdirectory(lxqt-config-file-associations)
 add_subdirectory(lxqt-config-appearance)
+add_subdirectory(lxqt-config-file-associations)
```

Comment:
Little note: when you are reordering the lines... here it seems that you are also removing some stuff on first sight. If you want the reordering happen I would suggest to do the reordering as separate commit.